media/server/gstplayer/include/GenericPlayerContext.h

```diff
@@ -130,6 +130,12 @@ struct GenericPlayerContext
      */
     double pendingPlaybackRate{kNoPendingPlaybackRate};
 
+    /**
+     * @brief Pending immediate output for MediaSourceType::VIDEO, it's possible we'll have to add
```

Comment:
I would remove the bit 'it's possible we'll have to add a "pendingImmediateOutputForAudio" in the future.'

I don't think immediate-output is supported on any of the Audio sinks, if it ever is we can sort it then.